### PR TITLE
Update Arm and Bicep templates for recommended metric alerts

### PR DESCRIPTION
**Pull request by Sohamdg081992**
This change updates Arm and Bicep templates for recommended metric alerts

### CODE REVIEW

File: GeneratedMonitoringArtifacts/Default/DefaultAlerts.json [hunk 1]
```diff
@@ -1,438 +0,0 @@
-{
```
Comment (vishiy) [inline GeneratedMonitoringArtifacts/Default/DefaultAlerts.json:1]: i see this deleted file is currently referenced by https://aka.ms/azureprometheus-communityalerts from documentation...

File: AddonBicepTemplate/AzureMonitorAlertsProfile.bicep [hunk 2]
```diff
@@ -1,534 +0,0 @@
-param monitorWorkspaceName string
```
Comment (vishiy) [inline AddonBicepTemplate/AzureMonitorAlertsProfile.bicep:1]: this deleted file is currently referenced by https://aka.ms/azureprometheus-alerts-bicep in our docs....

File: GeneratedMonitoringArtifacts/Default/DefaultAlerts.json (hunk 1 above)
Comment (Sohamdg081992) [reply GeneratedMonitoringArtifacts/Default/DefaultAlerts.json:1]: my bad, I deleted this by mistake. I will add them back and will delete after updating the public doc to point to the new file. Thanks for catching!

File: AddonBicepTemplate/AzureMonitorAlertsProfile.bicep (hunk 2 above)
Comment (Sohamdg081992) [reply AddonBicepTemplate/AzureMonitorAlertsProfile.bicep:1]: my bad, I deleted this by mistake. Adding back.

File: GeneratedMonitoringArtifacts/Default/DefaultAlerts.json (hunk 1 above)
Comment (Sohamdg081992) [reply GeneratedMonitoringArtifacts/Default/DefaultAlerts.json:1]: I reset soft the git commit to put the files back and now the branch got locked out and won't let me push. Created a new pr https://github.com/Azure/prometheus-collector/pull/731 from a new branch. Sorry about that.

**Pull request closed without merging**